gpt4all-chat/qml/ModelDownloaderDialog.qml

```diff
@@ -187,10 +187,19 @@ MyDialog {
                                         color: installButton.hovered ? theme.backgroundDark : theme.backgroundDarkest
                                     }
                                     onClicked: {
-                                        if (openaiKey.text === "")
-                                            openaiKey.showError();
-                                        else
-                                            Download.installModel(filename, openaiKey.text);
+                                        if (isOpenAICompatible) {
+                                            if (openaiBase.text === "")
+                                                openaiBase.showError();
+//                                            else if (openaiModel.text === "")
+//                                                openaiBase.showError();
```

Comment:
We should do something with this commented-out code - either remove it if it isn't needed, or uncomment it if it is.

Reply:
You're right.

Most implementations force the model name to match betwen the client and the server.
But In my own implementation, I prefer to leave this blank because I plan to serve only one model per instance, so the model name is not important.

So it is probably better to let the user choose.